src/server/generic_family_test.cc

```diff
@@ -4,6 +4,9 @@
 
 #include "server/generic_family.h"
 
+#include "absl/strings/str_format.h"
```

Comment:
you do not need to use a real clock for these tests. please take a look at AdvanceTime helper function. This is how we control time in our unit tests